cmd/operator/main.go

```diff
@@ -325,16 +355,80 @@ func run(fs *flag.FlagSet) int {
 		}
 	}
 
-	ao, err := alertmanagercontroller.New(ctx, restConfig, cfg, logger, r, canReadStorageClass, eventRecorderFactory)
+	alertManagerSupported, err := checkPrerequisites(
```

Comment:
```suggestion
	alertmanagerSupported, err := checkPrerequisites(
```

Reply:
I fixed this nit in the last commit. CI says `TestAllNS/y/PromRulesExceedingConfigMapLimit` fails this time.

This test passes locally for me. And it doesn't make sense that a typo fix could fail an e2e test. Could you reactivate CI to see if it's flaky?

Reply:
I just realized I didn't rebase to the latest main. Let me rebase to see how it goes.

Reply:
After rebased everything is green now. Sorry for making noise...

Reply:
I think we have a few flaky tests and that was one of them, don't worry 😅